requirements/base.txt

```diff
@@ -26,6 +26,8 @@ jsonpatch==1.7
 networkx==1.9
 lxml==3.3.5
 drf-extensions==0.2.5
+httpsig==1.1.0
+djangorestframework-httpsignature==0.2.1
```

Comment:
is this the only available DRF package? Seems quite new so maybe not mature enough.